phys2bids/cli/run.py

```diff
@@ -89,20 +89,18 @@ def _get_parser():
                           dest='tr',
                           type=float,
                           help='TR of sequence in seconds.',
-                          default=1)
+                          default=0)
     optional.add_argument('-thr', '--threshold',
                           dest='thr',
                           type=float,
                           help='Threshold used for trigger detection.',
                           default=2.5)
-    optional.add_argument('-tbhd', '--table-header',
-                          dest='table_header',
+    optional.add_argument('-chnames', '--channel-names',
+                          dest='ch_name',
                           nargs='*',
                           type=str,
                           help='Columns header (for json file).',
-                          # #!# Has to go to empty list
-                          default=['time', 'respiratory_chest', 'trigger',
-                                   'cardiac', 'respiratory_CO2', 'respiratory_O2'])
+                          default=[''])
```

Comment:
Having the default as `['']` will mean that `options.ch_name` in the` phys2bids.py` module will always evaluate to True. I think we should just leave the default off (which means, if not provided, it will default to None), and then check for equivalency to NoneType in `phys2bids.py`. I'll make a comment in the relevant spot!

Reply:
Doesn't it interfere with the check that the parser is doing on the type of input we expect?